src/Raven.Server/Documents/Handlers/SubscriptionsHandler.cs

```diff
@@ -341,7 +341,7 @@ await using (var writer = new AsyncBlittableJsonTextWriter(context, ResponseBody
                         [nameof(SubscriptionState.Disabled)] = x.Disabled,
                         [nameof(SubscriptionState.LastClientConnectionTime)] = x.LastClientConnectionTime,
                         [nameof(SubscriptionState.LastBatchAckTime)] = x.LastBatchAckTime,
-                        ["Connection"] = GetSubscriptionConnectionJson(x.Connection),
+                        ["Connections"] = GetSubscriptionConnectionsJson(x.Connections),
                         ["RecentConnections"] = x.RecentConnections?.Select(r => new DynamicJsonValue()
```

Comment:
Cannot do that, this is a breaking change.
You need to add the _first_ subscription as well as _all_ subscription in a new property

Reply:
From what we can tell the client doesn't use the `Connection` property. It parses only the `SubscriptionState`.. 
So as far as I can tell no breaking changes to the C# client... but we can keep both if needed

Reply:
We may have users access this directly from the JSON, assume that this is in use.

Reply:
Done